src/PythonMigrationViewExtension/Controls/VisualDifferenceViewer.xaml

```diff
@@ -0,0 +1,74 @@
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        xmlns:diffplex="clr-namespace:DiffPlex.Wpf.Controls;assembly=DiffPlex.Wpf"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+        Title="Migration Assistant"
+        WindowStartupLocation="CenterOwner"
+        Background="#272822">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="44" />
+        </Grid.RowDefinitions>
+
+        <diffplex:DiffViewer x:Name="DiffView"
+                             OldTextHeader="Old"
+                             NewTextHeader="New"
+                             FontFamily="Consolas"
+                             FontSize="10pt"
+                             Background="#272822"
+                             Foreground="#F8F8F2"
+                             HeaderForeground="#808080">
+        </diffplex:DiffViewer>
+
+            <Grid HorizontalAlignment="Stretch" Grid.Row="1" Background="Black">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="100" />
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
+            <StackPanel Orientation="Horizontal"
+                        HorizontalAlignment="Left"
+                        Grid.Column="0">
+                <Button  x:Name="DiffButton"
+                         Content="Switch Mode"
+                         Click="DiffButton_Click"
+                         Style="{StaticResource STextButton}"
+                         HorizontalAlignment="Right"
+                         VerticalAlignment="Bottom"
+                         Margin="1" />
+            </StackPanel>
+            <StackPanel Orientation="Horizontal"
+                        HorizontalAlignment="Right"
+                        Grid.Column="1">
+                <Button  x:Name="AcceptButton"
+                         Content="Accept Changes"
+                         Style="{StaticResource STextButton}"
+                         HorizontalAlignment="Right"
+                         VerticalAlignment="Bottom"
+                         Margin="1" />
+                <Button  x:Name="RejectButton"
+                         Content="Reject Changes"
+                         Style="{StaticResource STextButton}"
+                         HorizontalAlignment="Right"
+                         VerticalAlignment="Bottom"
+                         Margin="1" />
```

Comment:
we can define a common style and use that on the buttons that are the same

Reply:
I dont think thats possible as we are already setting the `Style` on the button, so i cant add another style to it as the one set directly on the button will overwrite, at least i havent been able to do it like this.

Reply:
you can combine them, will send you a link to an internal repo example